plane/source/docs/qland-mode.rst

```diff
@@ -0,0 +1,45 @@
+.. _qland-mode:
+
+==========
+QLAND Mode
+==========
+
+QLAND Mode attempts to bring the QuadPlane straight down descending to :ref:`Q_LAND_FINAL_ALT <Q_LAND_FINAL_ALT>` at :ref:`Q_WP_SPEED_DN <Q_WP_SPEED_DN>` until it reaches :ref:`Q_LAND_FINAL_ALT <Q_LAND_FINAL_ALT>`, at which point it continues to descend at :ref:`Q_LAND_SPEED <Q_LAND_SPEED>` until landing.
```

Comment:
should note that the position for the land is taken from the moment that QLAND is engaged